styles/index.css

```diff
@@ -0,0 +1,171 @@
+html {
+    background: linear-gradient(5deg, rgb(20, 108, 148) 0%, rgb(25, 167, 206) 35%, rgb(175, 211, 226) 100%);
+    font-family: monospace;
+    border: 10px solid rgb(60, 72, 107);
```

Comment:
I'm obsessed with the cool gradient background! Great job researching and implementing this feature!